.github/workflows/tag_release.yml

```diff
@@ -33,3 +33,12 @@ jobs:
       env:
         GOVERSION: ${{ env.GOVERSION }}
         GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+    - name: Create Sentry release
+      uses: getsentry/action-release@v1
+      env:
+        SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+        SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+      with:
+        environment: 'production'
+        version: '${{ github.ref_name }}'
```

Comment:
I tested this [here](https://github.com/Integralist/actions-testing/blob/main/.github/workflows/tag_release.yaml) and can confirm that `github.ref_name` will be the tag (e.g. `1.0.0`)